cmd/incus/admin_recover.go

```diff
@@ -10,7 +10,7 @@ import (
 	"golang.org/x/text/cases"
 	"golang.org/x/text/language"
 
-	"github.com/lxc/incus/v6/client"
+	incus "github.com/lxc/incus/v6/client"
```

Comment:
Looks like you ran goimports or something like that which has a tendency to go mess with our import names, can you remove all of those from your commit?